FtpClient/FtpCli/Program.cs

```diff
@@ -2,11 +2,59 @@
 
 namespace FtpCli
 {
-    class Program
+    public class Program
     {
+        public string server;
+        public string user;
+        public string[] commands;
+
+        public Program() {
+            server = "";
+            user = "";
+            commands = null;
+        }
+
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            Program FTP = new Program();
+
+            FTP.parseArgs(args);
+        }
+
+
+        // parses command line arguments
+        public void parseArgs(string[] args) {
+
+            // if an incorect number of parameters were passed
+            // print usage details and prompt user again
+            string usage = "servername username [cmd, args, ...]";
+            while (args.Length < 2) {

```

Comment:
This is not the way the application is working according to the other open PR right now: https://github.com/cs410agile-group3/FTPClient/pull/50

Reply:
I branched this from Michelle's earlier work before she refactored. Should be fixed on merge.